examples/fc_example/fc_example.ino

```diff
@@ -409,11 +415,12 @@ struct myTheme: public colorMapMono {
 struct colorMapGreen: public colorMap {
   // baseColor is used with just changes in brightness for a largely monochromatic theme.
   // You can change baseColor from green to change all these colors at once.
-  // When referring to a placeholder, you must use the FC_REF_MAP_COLOR() function.
-  FC_MAP_COLOR(FC_REF_MAP_COLOR(baseColor), green)
```

Comment:
`FC_REF_MAP_COLOR` must only be used when referencing the color-function with the given name (see the definition of `FC_REF_MAP_COLOR`).